src/app/extensions.cpp

```diff
@@ -25,6 +25,7 @@
 #include "base/file_handle.h"
 #include "base/fs.h"
 #include "base/fstream_path.h"
+#include "base/log.h"
```

Comment:
I know it looks good here...but **I think** that it has to be the last include of all the includes...so, I would move this under the <string> include.

Reply:
Nice catch!

Reply:
:+1: I forgot about the `#undef ERROR` in `base/log.h`, and in several files `base/log.h` is not included as the last header file. Anyway in the future we might rename the `ERROR` macro to avoid clashes with other libraries (or we could just use other logging library, like https://github.com/gabime/spdlog and remove our functions, but it's something for the future). I'll merge this.